addons/event/models/event_registration.py

```diff
@@ -51,6 +51,15 @@ class EventRegistration(models.Model):
         ('open', 'Confirmed'), ('done', 'Attended')],
         string='Status', default='draft', readonly=True, copy=False, tracking=True)
 
+    def default_get(self, fields):
```

Comment:
Obviously not for stable but I can't really figure out why we're not using the utm.mixin on this model to handle that automatically?
(Would require field name update though).